### PR TITLE
implement WordPress coding standards (#7)

### PR DESCRIPTION
**Pull request by ChristofLee**
This is an example of we can use PHPCS and PHPCBF to format our code and standardise it.

Solves #7.

### CODE REVIEW

**Comment by warengonzaga** [comment]
@ChristofLee don't forget to update this.

**Comment by ChristofLee** [comment]
Was awaiting your approval on this code style. I'll take this as your happy with it and I will update this PR asap.

**Comment by warengonzaga** [comment]
> Was awaiting your approval on this code style. I'll take this as your happy with it and I will update this PR asap.

your PR is currently in draft... ask me for a review if you are ready.

File: shortcodes-wp.php [hunk 1]
```diff
@@ -1,38 +1,87 @@
...
 
 /**
-* WP Update Your Footer
-*/
+ * WP Update Your Footer
+ */
 
-// prevent direct access
-defined( 'ABSPATH' ) or die( "Restricted Access!" );
+// Prevent direct access.
```
Comment (warengonzaga) [inline shortcodes-wp.php]: Keep this single comments in lowercase.

File: shortcodes-wp.php [hunk 2]
```diff
@@ -1,38 +1,87 @@
...
-    $firstname = $user->first_name;
-    $display_firstname = $firstname;
+function user_firstname( $atts ) {
+	$user              = wp_get_current_user();
+	$firstname         = $user->first_name;
+	$display_firstname = $firstname;
 
-    // display output
-    return $display_firstname;
+	// Display output.
```
Comment (warengonzaga) [inline shortcodes-wp.php]: Same here, keep this in lowercase for single comments.

File: shortcodes-wp.php [hunk 3]
```diff
@@ -1,38 +1,87 @@
...
-function user_lastname($atts) {
-    $user = wp_get_current_user();
-    $lastname = $user->last_name;
-    $display_lastname = $lastname;
+function user_lastname( $atts ) {
+	$user             = wp_get_current_user();
+	$lastname         = $user->last_name;
+	$display_lastname = $lastname;
 
-    // display output
```
Comment (warengonzaga) [inline shortcodes-wp.php:32]: Another one!

**Review by warengonzaga — CHANGES_REQUESTED** [review]
Overall good for me however please revert back the lowercase for single line comments.

**Comment by ChristofLee** [comment]
Contains fixes from other branches and is not isolated to this one issue. I will start again and make a new PR.

**Pull request closed without merging**